### PR TITLE
chore(tutorial): update cf login instructions for Vue, Angular

### PR DESCRIPTION
**Pull request by tw15egan**
Updates the Vue and Angular `cf login` command to the updated version present in the React tutorial steps

### CODE REVIEW

**Review by emyarod — APPROVED** [review]
looks good to me

**Review by mattrosno — APPROVED** [review]
I completed step 5 just fine yesterday, so if this makes Angular and Vue match the CF instructions for the React tutorial, LGTM.

https://github.com/carbon-design-system/carbon-tutorial/pull/6531

File: src/pages/developing/angular-tutorial/step-5.mdx [hunk 1]
```diff
@@ -234,17 +234,23 @@ on IBM Cloud. For example, in the case of serving static files, you can ignore
...
 multiple manifest files that target different environments, it's good practice
 to specify the file.
 
-_Note: To successfully deploy, you might need to update the region code (e.g.
-`api.[REGION].bluemix.net`).
-[Learn more.](https://developer.ibm.com/answers/answers/166990/view.html)_
+<InlineNotification>
+
+**Note:** This step assumes your spaces are in the US South region. To
+successfully deploy, you might need to update the region code (e.g.
```
Comment (janchild) [inline src/pages/developing/angular-tutorial/step-5.mdx:248]:
```suggestion
successfully deploy, you might need to update the region code (for example,
```

File: src/pages/developing/angular-tutorial/step-5.mdx [hunk 2]
```diff
@@ -234,17 +234,23 @@ on IBM Cloud. For example, in the case of serving static files, you can ignore
...
+`api.[REGION].cf.cloud.ibm.com`) to the region where your spaces were created,
+or create a space in the US South region.
+[Learn more.](https://cloud.ibm.com/docs/overview?topic=overview-whatsnew&origin_team=T02M79KSB#new-cloud-foundry-api-endpoints)
```
Comment (janchild) [inline src/pages/developing/angular-tutorial/step-5.mdx:251]:
```suggestion
[Learn more](https://cloud.ibm.com/docs/overview?topic=overview-whatsnew&origin_team=T02M79KSB#new-cloud-foundry-api-endpoints)
```

File: src/pages/developing/vue-tutorial/step-5.mdx [hunk 3]
```diff
@@ -235,20 +235,26 @@ IBM Cloud. For example, in the case of serving static files, you can ignore
...
 multiple manifest files that target different environments, it's good practice
 to specify the file.
 
-_Note: To successfully deploy, you might need to update the region code (e.g.
-`api.[REGION].bluemix.net`).
-[Learn more.](https://developer.ibm.com/answers/answers/166990/view.html)_
+<InlineNotification>
+
+**Note:** This step assumes your spaces are in the US South region. To
+successfully deploy, you might need to update the region code (e.g.
```
Comment (janchild) [inline src/pages/developing/vue-tutorial/step-5.mdx:252]:
```suggestion
successfully deploy, you might need to update the region code (for example, 
```

File: src/pages/developing/vue-tutorial/step-5.mdx [hunk 4]
```diff
@@ -235,20 +235,26 @@ IBM Cloud. For example, in the case of serving static files, you can ignore
...
+`api.[REGION].cf.cloud.ibm.com`) to the region where your spaces were created,
+or create a space in the US South region.
+[Learn more.](https://cloud.ibm.com/docs/overview?topic=overview-whatsnew&origin_team=T02M79KSB#new-cloud-foundry-api-endpoints)
```
Comment (janchild) [inline src/pages/developing/vue-tutorial/step-5.mdx:255]:
```suggestion
[Learn more](https://cloud.ibm.com/docs/overview?topic=overview-whatsnew&origin_team=T02M79KSB#new-cloud-foundry-api-endpoints)
```

**Review by janchild — COMMENTED** [review]
@tw15egan Some small content tweaks required to keep in line with style. Thank you!

**Pull request merged**